gammapy/data/gti.py

```diff
@@ -78,21 +96,20 @@ def create(cls, start, stop, reference_time="2000-01-01"):
             the reference time to use in GTI definition
         """
         reference_time = Time(reference_time)
+        reference_time.format = "mjd"
 
-        if isinstance(start, Time):
-            start = (start - reference_time).to(u.s)
+        if not isinstance(start, Time):
+            start = reference_time + u.Quantity(start)
 
-        if isinstance(stop, Time):
-            stop = (stop - reference_time).to(u.s)
+        if not isinstance(stop, Time):
+            stop = reference_time + u.Quantity(stop)
 
-        start = u.Quantity(start, ndmin=1)
-        stop = u.Quantity(stop, ndmin=1)
-        meta = time_ref_to_dict(reference_time)
-        table = Table({"START": start.to("s"), "STOP": stop.to("s")}, meta=meta)
-        return cls(table)
+        table = Table({"START": np.atleast_1d(start), "STOP": np.atleast_1d(stop)})
+
+        return cls(table, reference_time=reference_time)
 
     @classmethod
-    def read(cls, filename, hdu="GTI"):
+    def read(cls, filename, hdu="GTI", format="gadf"):
```

Comment:
The hdu name "GTI" is for the GADF format... Should we assume that the hdu name is independent of the format?

Reply:
I am not sure the hduname is really imposed by the gadf.